notebooks/SMAP/3.0 SMAP Quality Flags.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<p>Use dataset paths instead of keys and indexed lists</p><pre class="ql-syntax" spellcheck="false">f = h5py.File(filename, 'r')
sm_data = f['Soil_Moisture_Retrieval_Data_AM/soil_moisture'][:]
surf_flag = f['Soil_Moisture_Retrieval_Data_AM/surface_flag'][:]
</pre><p><code>[:]</code> works for complete data arrays. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>